Examples/NotificationService.cs

```diff
@@ -1,15 +1,37 @@
 public class NotificationService()
 {
-    public ... Get(string type)
+    public void Get(string type)
```

Comment:
same as previous comment.

Reply:
you have to change your return type, since we have a method to Get a new instance.